virtual_workstation_setup.sh

```diff
@@ -179,10 +214,15 @@ setup_gdb() {
     pushd "$workdir"
         if [[ -d 'Boost-Pretty-Printer' ]]; then
             echo "'Boost-Pretty-Printer' dir exists; skipping setup"
-            return
+        else
+            git clone git@github.com:ruediger/Boost-Pretty-Printer.git
```

Comment:
💅  checkout a specific commit or version to avoid third-party upstream bugs from breaking our workflow.

Reply:
Leaving this alone as this is what we've always done here. Even previous onboarding docs don't list a specific commit.